docs/entities/renderer.md

```diff
@@ -0,0 +1,584 @@
+    private final ModelPart head;
+    
+    // The ModelPart passed here is the root of our baked model.
+    // We will get to the actual baking in just a moment.
+    public MyEntityModel(ModelPart root) {
+        // Store the head part for use below.
+        this.head = root.getChild("head");
+    }
+
+    public static LayerDefinition createBodyLayer() {...}
+
+    // Use this method to update the model rotations, visibility etc. from the render state. If you change the
+    // generic parameter of the EntityModel superclass, this parameter type changes with it.
+    @Override
+    public void setupAnim(MyEntityRenderState state) {
+        // Calling super to reset all values to default.
+        super.setupAnim(state);
+        // Change the model parts.
+        head.visible = state.myBoolean();
+        head.xRot = state.myXRotation();
+        head.yRot = state.myYRotation();
+        head.zRot = state.myZRotation();
+    }
+}
+```
+
+Now that our model is able to properly receive a baked `ModelPart`, we can create our `RenderLayer` subclass and use it for baking the `LayerDefinition` like so:
+
+```java
+// The generic parameters need the proper types you used everywhere else up to this point.
+public class MyRenderLayer extends RenderLayer<MyEntityRenderState, MyEntityModel> {
+    private final MyEntityModel model;
+    
+    // Create the render layer. The renderer parameter is required for passing to super.
+    // Other parameters can be added as needed. For example, we need the EntityModelSet for model baking.
+    public MyRenderLayer(MyEntityRenderer renderer, EntityModelSet entityModelSet) {
+        super(renderer);
+        // Bake and store our layer definition, using the ModelLayerLocation from back when we registered the layer definition.
+        // If applicable, you can also store multiple models this way and use them below.
+        this.model = new MyEntityModel(entityModelSet.bakeLayer(MY_LAYER));
+    }
+
+    @Override
+    public void render(PoseStack poseStack, MultiBufferSource bufferSource, int packedLight, MyEntityRenderState renderState, float yRot, float xRot) {
+        // Render the layer here. We have stored the entity model in a field, you probably want to use it in some way.
+    }
+}
+```
+
+### Adding a Render Layer to an Entity Renderer
+
+Finally, to tie it all together, we can add the layer to our renderer (which, if you remember, now needs to be a living renderer) like so:
+
+```java
+// Plugging in our custom render state class as the generic type.
+// Also, we need to implement RenderLayerParent. Some existing renderers, such as LivingEntityRenderer, do this for you.
+public class MyEntityRenderer 
+        extends LivingEntityRenderer<MyEntity, MyEntityRenderState>
+        implements RenderLayerParent<MyEntityRenderState, MyEntityModel> {
+    public MyEntityRenderer(EntityRendererProvider.Context context) {
+        super(context);
```

Comment:
super for `LivingEntityRenderer` takes in the context, parent model, and the shadow radius.